pandas/core/reshape/reshape.py

```diff
@@ -1054,19 +1054,20 @@ def get_empty_frame(data) -> DataFrame:
 
 
 def from_dummies(
-    data,
+    data: DataFrame,
     to_series: bool = False,
     prefix_sep: str | list[str] | dict[str, str] = "_",
     columns: None | list[str] = None,
     dropped_first: None | str | list[str] | dict[str, str] = None,
+    fillna: None | bool = None,
```

Comment:
rather than adding an extra argument, I'd say just throw an error if there's a nan and the user can fill it before calling from_dummies